tests/cypress/integration/wp-module-support/EventsApi.cy.js

```diff
@@ -18,7 +18,11 @@ export const APIList = {
 	basic_info:
 		'/index.php?rest_route=%2Fnewfold-onboarding%2Fv1%2Fevents%2Fbatch&flow=wp-setup&_locale=user',
 	basic_info_ecomm:
-		'/index.php?rest_route=%2Fnewfold-onboarding%2Fv1%2Fevents%2Fbatch&flow=ecommerce&_locale=user'
+		'/index.php?rest_route=%2Fnewfold-onboarding%2Fv1%2Fevents%2Fbatch&flow=ecommerce&_locale=user',
```

Comment:
we are using the same API for general and ecomm flow, so I think it's better to have only 2 APIs, one for general and one for ecomm.
also, please update where the APIs have been used.

Reply:
Done